node_modules/oae-admin/tenants/js/tenants.js

```diff
@@ -350,43 +309,65 @@ define(['jquery', 'oae.core', 'jquery.jeditable'], function($, oae) {
         };
 
         /**
-         * Render the list of tenants
+         * Initialize a new infinite scroll for the tenant list
+         *
+         * @param  {Function}   postRenderer                    A function to invoke each time new items are added to the list
```

Comment:
Rather than passing in the `postRenderer`, we can just define it in this function